### DISCUSSION TITLE
How to prevent app from stealing focus?

### DISCUSSION THREAD

**Discussion by albertsj1** [asmvik/yabai#1500] [Q&A]
I've been having an issue with Amazon Chime stealing the focus.  This happens when there is an active Chime call.  It doesn't seem to matter if someone is sharing their desktop or not.  During any active Chime call, if I have the window for that Chime call focused and then try to switch to a different app, the focus bounces back to the active Chime call.
I found an [old closed issue 261](https://github.com/koekeishiya/yabai/issues/261) that asks about the same issue with WebEx, but I don't really understand the couple of recommendations are mentioned in that issue.

Does anyone have any suggestions to prevent this type of focus stealing?

Thanks

**Comment by AlJohri** [comment]
Potentially related: https://github.com/koekeishiya/yabai/discussions/2183

**Reply by albertsj1** [reply]
Thanks for the reply.  I've stopped using Chime, so this isn't an issue for me any longer.
